cmake/developer_package/clang_format/clang_format_check.cmake

```diff
@@ -7,12 +7,13 @@ file(REMOVE "${OUTPUT_FILE}")
 execute_process(COMMAND ${CLANG_FORMAT} -style=file -output-replacements-xml ${INPUT_FILE}
     OUTPUT_VARIABLE STYLE_CHECK_RESULT
     )
-# Display the cpplint output to console (to parse it form IDE)
-message("${output}")
+
 file(WRITE "${OUTPUT_FILE}" "${STYLE_CHECK_RESULT}")
 
 if(NOT SKIP_RETURN_CODE)
     if("${STYLE_CHECK_RESULT}" MATCHES ".*<replacement .*")
-        message(FATAL_ERROR "[clang-format] Code style check failed for : ${INPUT_FILE}")
+        # Display the style check output to console in case an issue
+        message("Style check result: ${STYLE_CHECK_RESULT}")
```

Comment:
Why did you add these changes to cmake config? If I remember right the value of `${STYLE_CHECK_RESULT}` contains a lot of information and it is not user-friendly to read

Reply:
the previous message wasn't useful to it goes to the nonexistence file. I added it to get a little bit of info about clang-format issues.

Reply:
remove it at all.